src/server/server.js

```diff
@@ -197,6 +197,11 @@ Server.prototype._onSentryDown = function (sentryId) {
   var started = Date.now()
 
   nonblocking(presences).forEach(function (presence) {
+    // check if presence was destroyed
+    if (!presence || !presence.manager || !presence.manager.store) {
```

Comment:
there's a destroying flag on the manager which is completely useless. Use that here.